bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -536,7 +536,8 @@ void BebopDriverNodelet::AuxThread()
       if (bebop_ptr_->ardrone3_pilotingstate_positionchanged_ptr)
       {
         gps_state_ptr = bebop_ptr_->ardrone3_pilotingstate_positionchanged_ptr->GetDataCstPtr();
-        if ((gps_state_ptr->header.stamp - last_gps_time).toSec() > util::eps)
+        //if ((gps_state_ptr->header.stamp - last_gps_time).toSec() > util::eps)
```

Comment:
Sorry for my delayed feedback and thanks for making all the changes. I will merge it in very soon. Please also remove this commented line. Also, please consider adding your name to the following file as part of this pull request: https://github.com/AutonomyLab/bebop_autonomy/blob/indigo-devel/docs/contribute.rst